### DISCUSSION TITLE
New Bug (as of 0700 GMT): Dislikes Patched By Youtube (Edit: Resolved?)

### DISCUSSION THREAD

**Discussion by rohitchaoji** [Anarios/return-youtube-dislike#505] [General]
Using Firefox 97.0.2

As of 0700 GMT, Youtube seems to have patched whatever allowed the plugin to display the Dislike count. Now I can only see the the text saying "DISLIKE" rather than the dislike count. It was working fine about 15 minutes ago so it must be very recent.

Edit: My apologies but it seems to be a loading issue with Firefox, where the plugin didn't load correctly and did not display the Dislike counts. It is now displaying correctly as intended. Any way to delete this thread?

**Comment by Anarios** [comment]
Should be good now
